Update pr-wranglers.md (#45443)
kubernetes/website: content/en/docs/contribute/participate/pr-wranglers.md

    - Using style fixups as good first issues is a good way to ensure a supply of easier tasks
      to help onboard new contributors.

{{< note >}}

> PR Wrangler duties apply to English language PRs. Localization teams have their own processes and teams for reviewing their language PRs. However if you have time during your shift, its often helpful to make sure language PRs are labeled correctly, review small non-language dependent PRs (like a link update), or tag reviewers or contributors in long running PRs (ones opened more than 6 months ago and have not been updated in a month or more). 

Reviewer: non-blocking nit: `its` is possessive
```suggestion
PR Wrangler duties apply to English language PRs. Localization teams have their own processes and teams for reviewing their language PRs. However if you have time during your shift, it's often helpful to make sure language PRs are labeled correctly, review small non-language dependent PRs (like a link update), or tag reviewers or contributors in long running PRs (ones opened more than 6 months ago and have not been updated in a month or more). 
```

> PR Wrangler duties apply to English language PRs. Localization teams have their own processes and teams for reviewing their language PRs. However if you have time during your shift, its often helpful to make sure language PRs are labeled correctly, review small non-language dependent PRs (like a link update), or tag reviewers or contributors in long running PRs (ones opened more than 6 months ago and have not been updated in a month or more). 

Reviewer 2: ```suggestion
PR wrangler duties do not apply to localization PRs (non-English PRs). Localization teams have their own processes and teams for reviewing their language PRs. However, if you have time during your shift, it's often helpful to ensure language PRs are labeled correctly, review small non-language dependent PRs (like a link update), or tag reviewers or contributors in long-running PRs (ones opened more than 6 months ago and have not been updated in a month or more).
```
Hi @a-mccarthy 
I think the phrase "PR Wrangler duties apply to English language PRs" could be clarified further. The reason is that the current duties of a PR Wrangler apply not only to English language PRs but also to PRs that are independent of language (for example, workflow, site-configuration, scripts, etc.). Therefore, stating that these duties do not apply to localization PRs might make it clearer.

I'm not sure about the blog area. I haven't checked whether blog PRs are an exception to PR Wrangler duties or not.

Author: good point @seokho-son, i've added in your suggestions.

> {{< /note >}}

Reviewer 3: Please wrap this long line as we did for other paragraphs.
